src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -529,6 +534,11 @@ public void testUnsuccessfulCanReadFileCheck(final File file) {
     @Test
     public void testCreateDirectory() throws IOException {
 
+        // see https://github.com/eclipse/jetty.project/issues/8549
+        if (isGATKDockerContainer()) {
```

Comment:
There wasn't one before, but it looks like there has been some activity since I last looked at this. I'll add this to my "optional code review changes" commit and see if I can address it once all the tests are passing.

Reply:
I was able to remove many, but not all of these conditionals. I think there are 8 left.